fhir/com.b2international.snowowl.fhir.core/src/com/b2international/snowowl/fhir/core/request/valueset/FhirValueSetExpandRequest.java

```diff
@@ -244,26 +246,37 @@ private ValueSet computeFhirValueSetUsingUrl(ServiceProvider context, String url
 		Concepts concepts = req.buildAsync().execute(context);
 		
 		expansion
-			.total(concepts.getTotal())
-			.after(concepts.getSearchAfter());
+			.setTotal(concepts.getTotal())
+			.setUserData("after", concepts.getSearchAfter());
```

Comment:
Shouldn't we store these magic words in user data somewhere permanent?

Reply:
Extension is the way probably, we will consider changing it to something else. Unfortunately in order to be backward compatible we might need to inject it to the top level JSON object anyway.

Reply:
Replaced userData with custom Extension value in commit https://github.com/b2ihealthcare/snow-owl/pull/1323/commits/a7040c5d749db49de55f095c4121cd20cae4094b.